### PR TITLE
adding prow propsal .... new PR with signoff

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

I re-made the PR since I re-wrote history on rebase, and didn't want to clutter it up how to use DCO signing.

### CODE REVIEW

File: proposals/20200915-prow.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Horologium
+- Tide (will look into difficulty)
+- Spyglass
+- Crier
+- Testgrid
+- Triage
+- Deck
+
+Open investigation items
+- Slack reporter to Falco K8s channel
```
Comment (leodido) [inline proposals/20200915-prow.md:48]: I'm okay with this if we report to another #falco-something channel, not the main one.

File: proposals/20200915-prow.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- Prom/Grafana for Prow monitoring
```
Comment (leodido) [inline proposals/20200915-prow.md:49]: 💯 for me.

Not a top priority, though.

**Review by leodido — APPROVED** [review]
Definitely a good initial plan for me!

/hold
waiting also for @fntlnz thoughts

**Review by fntlnz — APPROVED** [review]

**Pull request merged**